### PR TITLE
Release information for 4.09.0

### PR DESCRIPTION
**Pull request by Octachron**
This PR adds a page for the upcoming release of 4.09.0 . Please tell me if I have missed anything. This should not be merged yet.

### CODE REVIEW

File: site/releases/4.09.0.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+
+- 4.09.0 — Official release 4.09.0
+- 4.09.0+32bit - Official release 4.09.0, compiled in 32-bit mode
+  for 64-bit Linux and OS X hosts
+- 4.09.0+afl — Official release 4.09.0, with afl-fuzz instrumentation
+- 4.09.0+bytecode-only - Official release 4.09.0, without the
+  native-code compiler
+- 4.09.0+default-unsafe-string — Official release 4.09.0, without
+  safe strings by default
+  strings by default
```
Comment (lpw25) [inline site/releases/4.09.0.md]: duplicate line?

Comment (Octachron) [reply site/releases/4.09.0.md]: Fixed, thanks.

**Comment by pw374** [comment]
Please could you prefix the title by "WIP" until it's ready to be merge? :)

**Comment by Octachron** [comment]
Sure.

**Comment by Octachron** [comment]
This should be fine to squash and merge now. Or would you prefer that I do the squashing myself?

**Comment by Chris00** [comment]
No need.  Merge done.

**Pull request merged**